### PR TITLE
Greetings API

### PR DESCRIPTION
**Pull request by amejid**
In this PR, the following were implemented:-

- Created a greeting model
- Created an API route that provides a random greeting

### CODE REVIEW

File: config/routes.rb [hunk 1]
```diff
@@ -1,6 +1,3 @@
 Rails.application.routes.draw do
-  # Define your application routes per the DSL in https://guides.rubyonrails.org/routing.html
-
-  # Defines the root path route ("/")
-  # root "articles#index"
+  resources :greetings, only: [:index]
```
Comment (TedLivist) [inline config/routes.rb:2]: - [ ] When I try to fetch the greeting. I get this CORS policy error. Kindly fix that. You can visit the [CORS gem](https://github.com/cyu/rack-cors) and configure your backend to allow cross-origin requests.

  Basically, you need to:
  - Uncomment `gem 'rack-cors'` in your Gemfile and run `bundle install`
  - In the `config/initializers/cors.rb` file and uncomment the following code and remember to change "example.com" to "*":
    
    ```
    Rails.application.config.middleware.insert_before 0, Rack::Cors do
      allow do
        origins '*'
        resource '*', headers: :any, methods: [:get, :post, :patch, :put]
      end
    end
    ```
  - Restart your server

  image

**Review by TedLivist — CHANGES_REQUESTED** [review]
## STATUS: CHANGES REQUIRED ♻️ ♻️

Hi @amejid,

Good job so far!
✔️ Good documentation
However, there are some issues that you still need to work on to go to the next project but you are almost there!

## Required Changes ♻️
_Check the comments under the review._

Cheers and Happy coding!👏👏👏

Feel free to leave any questions or comments in the PR thread if something is not 100% clear. And please, remember to tag me in your question so I can receive the notification.

**_Please, do not open a new Pull Request for re-reviews. You should use the same Pull Request submitted for the first review, either valid or invalid unless it is requested otherwise._**

------
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561) you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._

File: config/routes.rb (hunk 1 above)
Comment (amejid) [reply config/routes.rb:2]: I will implement it accordingly.

**Comment by amejid** [comment]
Thanks for the constructive feedback.

**Review by rloterh — COMMENTED** [review]
Hi Abdelmejid, 

While you made a great effort in this project, unfortunately, I cannot proceed to review your code.

## Invalid Code Review Request

Kindly ensure that the ReadMe of this project (backend) has a link to the related frontend, and similarly, the ReadMe of the corresponding frontend should contain a link to this backend. Kindly fix the corresponding links in your ReadMe, and then request for another review.

Your Code Review Request will be marked as invalid in your Dashboard, so please submit a new one once you are ready 🙏

Cheers and Happy coding!👏👏👏

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.
Please, remember to tag me in your question so I can receive the notification.

**_Please, do not open a new Pull Request for re-reviews. You should use the same Pull Request submitted for the first review, either valid or invalid unless it is requested otherwise._**

--------

_Invalid Code Review Request **does not count** into the code reviews limit._

**Comment by amejid** [comment]
@rloterh The links to the corresponding project is present.
Screenshot from 2022-11-22 16-01-44
Screenshot from 2022-11-22 16-02-35

**Comment by rloterh** [comment]
@amejid 
Thanks for pointing this out. However, I think you placed the frontend link where it seems to be obscure, especially if a visitor is unaware that the link is placed under a foldable list item. I think you should place the link where it will be easier for visitors to your repo to find it.
image

Cheers! 😃

**Review by Davidosky007 — APPROVED** [review]
Hi @amejid 👋, 

Your project is complete!  There is nothing else to say other than... it's time to merge it :shipit: 
Congratulations! 🎉 

## Highlights
* No linter errors 👍 
* Well written PR description 👍 
* Good looking readme file 👍 
## Optional suggestions

_Every comment with the [OPTIONAL] prefix won't stop the approval of this PR. However, I strongly recommend you take them into account as they can make your code better. Some of them were simply missed by the previous reviewer and addressing them will improve your application._

Cheers and Happy coding!👏👏👏

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.
Please, remember to tag me in your question so I can receive the notification.

------
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561)  you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._

**Pull request merged**